spec/services/event_subscriber_translate_service_spec.rb

```diff
@@ -4,6 +4,17 @@
 require 'ostruct'
 
 describe Force::EventSubscriberTranslateService do
+  before do
+    allow(::UNLEASH).to receive(:is_enabled?) # rubocop:disable Style/RedundantConstantBase
+      .with('GoogleCloudTranslate')
+      .and_return(true)
+  end
```

Comment:
issue: Need to test the scenario where the flag is off, to bring up code coverage levels.